pset-iterators.js

```diff
@@ -161,6 +202,12 @@ console.log('Problem 6:')
 
 // Add your code below this line
 
+const firstRepublican = presidents.find((president) => {
+	return president.party === "Republican"
+})
+
+console.log(`The first Republican president was ${firstRepublican.president}.`)
+
```

Comment:
@cswebdesign Great job here using the find method to return the first republican president.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)